VCFv4.4.draft.tex

```diff
@@ -1701,6 +1706,40 @@ \subsection{Representing unspecified alleles and REF-only blocks (gVCF)}
 \end{flushleft}
 \normalsize
 
+
+\pagebreak
+\subsection{Representing copy number variation}
+\label{cnv}
+
+To encode copy number variation, VCF uses $<$CNV$>$, $<$DEL$>$ and $<$DUP$>$ symbolic structural variant alleles, CN INFO and FORMAT fields.
+
+Allele specific copy number is specified through a $<$CNV$>$ ALT allele for each distinct allelic copy number.
+POS and INFO SVLEN specify the genomic interval over which the copy number is defined.
+As with all symbolic structural variants, the starting position of the interval is the base immediately after POS.
+For example, a region on chr1 from position 101 to 300 (both inclusive) with allele-specific copy numbers of 1 and 2 can be represented as follows:
+
+\footnotesize
+\begin{verbatim}
+	chr1 100 . T <CNV>,<CNV> . . END=130;SVLEN=30,30;CN=1,2 GT:CN 1/2:3
+\end{verbatim}
+\normalsize
+
+All $<$CNV$>$ alleles in the same VCF record should have the same SVLEN.
+To eliminate ambiguity in variant interpretation when GT=0, all copy number haplotypes should be explicitly specified with $<$CNV$>$ alleles, even when copy number is 1.
```

Comment:
It doesn't seem like there will be any ambiguity for the interpretation of GT=0 when there is a symbolic SV ALT allele. It may be better to just stick with GT=0 for representing the copy neutral haplotype rather than introducing an alternative representation that would have the same meaning.

Reply:
GT=0 is only unambiguous if all ALT records are copy number records. I've updated the wording to allow GT=0 as CN=1 if all the ALT records are copy number claims (and made the stronger claim that copy number ALTs shouldn't be mixed with non-copy number ALT).